server/src/integration/java/org/opentosca/toscana/plugins/kubernetes/docker/image/DockerImageBuilderIT.java

```diff
@@ -44,12 +47,12 @@ public void testBuildShaImage() throws Exception {
     }
 
     private boolean isDockerAvailable() {
```

Comment:
Do we still need this at all?

Reply:
I would keep it. just to be save. When trying to run the integration tests with no docker daemon available this test will fail.